### PR TITLE
Added pure theme with zsh shell and Atom color scheme

### PR DESCRIPTION
**Pull request by shettykaran21**
Please merge my custom theme if you like it : )

### CODE REVIEW

File: pure.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# Pure
+
+### Theme setup
```
Comment (rjcarneiro) [inline pure.md:3]: Why do you have this section here? Any particular reason? I can't see the benefit...

File: pure.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+- Zsh shell
+- oh-my-zsh with pure theme
+- Terminal with Atom color scheme
+
+![Pure](images/pure.png)
+
+## Fonts
```
Comment (rjcarneiro) [inline pure.md:11]: It's missing a the `## Settings` section.

**Review by rjcarneiro — COMMENTED** [review]
Hi @shettykaran21 ,

Thanks for your pull request. Can you please update according to the comments?

Thanks in advance.

File: pure.md [hunk 3]
```diff
@@ -1,12 +1,20 @@
 # Pure
 
-### Theme setup
+![Pure](images/pure.png)
 
-- Zsh shell
-- oh-my-zsh with pure theme
-- Terminal with Atom color scheme
+### Settings
```
Comment (rjcarneiro) [inline pure.md:5]: This should be header second level `##` instead of `###`

**Review by rjcarneiro — COMMENTED** [review]
Lovely. Almost there.

Also, it's missing an entry in the main README.md file.

**Comment by rjcarneiro** [comment]
Thanks @shettykaran21 for your amazing contribution 🙏

**Comment by shettykaran21** [comment]
@rjcarneiro It was a pleasure 😊

**Pull request merged**